File: app/Models/barang.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class barang extends Model
{
    protected $table = 'barangs';
    protected $fillable = ['nama', 'kategori', 'stok', 'harga', 'modal', 'barcode', 'gambar_path'];
}

File: app/src/main/java/com/application/managerusahav2/data/model/response/Barang.kt
package com.application.managerusahav2.data.model.response

import com.google.gson.annotations.SerializedName

data class Barang(
    val id: Int,
    val nama: String,
    val kategori: String,
    val stok: Int,
    val harga: Double,
    val modal: Double,
    val barcode: String?,
    @SerializedName("gambar_path")
    val gambarPath: String?,
)
